app/views/admin/tags/show.html.haml

```diff
@@ -3,9 +3,12 @@
 
 - content_for :heading_actions do
   - if current_user.can?(:view_dashboard)
-    = l(@time_period.first)
-    = ' - '
-    = l(@time_period.last)
+    .time-period
+      = l(@time_period.first)
+      = ' - '
+      = l(@time_period.last)
```

Comment:
We probably want to consider using a partial for this, since we've this exact contents in many places.